lib/install/config/loaders/installers/elm.js

```diff
@@ -1,5 +1,9 @@
+const path = require('path')
+
+const elmSource = path.join(__dirname, '..', '..', '..')
```

Comment:
@bkuhlmann Right I see, I don't know much about elm that's why asked. In that case lets use `path.resolve('elm-package.json')`